docs/usage/types.md

```diff
@@ -824,6 +824,7 @@ The following arguments are available when using the `conbytes` type function
 - `to_lower: bool = False`: turns all characters to lowercase
 - `min_length: int = None`: minimum length of the byte string
 - `max_length: int = None`: maximum length of the byte string
+- `strict: bool = False`: controls type coercion
```

Comment:
Maybe put it before `strip_whitespace` like all the others above?

Reply:
I can change it.
But, I kept the order of arguments.
Also,  `constr` is written onsame rule.

https://github.com/samuelcolvin/pydantic/blob/37c37fd55eaec24094c8e8c162489a97e6d02c59/pydantic/types.py#L397-L406

https://github.com/samuelcolvin/pydantic/blob/37c37fd55eaec24094c8e8c162489a97e6d02c59/docs/usage/types.md#L809-L819

Reply:
Ah right! Since it's kwarg only methods, the order doesn't matter. But it's just a detail in the documentation I have no strong opinion x)

Reply:
Either way, the document is not consistent.
 I think we are better to create other PR to fix the document. If we need to change the order of the description.
 But, my opinion is not strong too :)